inlang/source-code/paraglide/paraglide-js/src/cli/commands/init.ts

```diff
@@ -22,6 +23,7 @@ export const initCommand = new Command()
 	.summary("Initializes inlang Paraglide-JS.")
 	.action(async () => {
 		consola.box("Welcome to inlang Paraglide-JS 🪂")
+		telemetry.capture({ event: "PARAGLIDE-JS init executed" })
 
 		await checkIfUncommittedChanges()
```

Comment:
I would recommend multiple events to have an insight if the init command worked: 

```
1. PARAGLIDE-JS init started
2. ... we can add various events here to build a funnel and see where people drop off
3. PARAGLIDE-JS init finished

Reply:
Will do!